swf/src/read.rs

```diff
@@ -261,6 +261,15 @@ impl<'a, 'b> BitReader<'a, 'b> {
         self.bits.read_bit()
     }
 
+    #[inline]
```

Comment:
I'd include docs that describe the difference between `read_ubits` and `read_ubits_ct` & elaborate on what `_ct` means. I'd do the same for `read_ubits` too and include information that it's better to use `read_ubits_ct` when applicable.

This also applies to `write_ubits_ct`